.github/workflows/reusable-workflow.yml

```diff
@@ -33,6 +36,10 @@ jobs:
       - name: Run e2e tests
         run: ${{ inputs.docker_compose_command }}
         env:
+          # conditionals based on github event
+          SYNPRESS_PROFILE: ${{ env.IS_EMERYNET_TEST == 'true' && 'daily-tests' || 'synpress' }}
+          CYPRESS_AGORIC_NET: ${{ env.IS_EMERYNET_TEST == 'true' && 'emerynet' || 'local' }}
+          # for docker-compose.yml
```

Comment:
I think `AGORIC_NET` is enough

Reply:
the `CYPRESS_` prefix is required to be able to use it inside our cypress tests https://docs.cypress.io/guides/guides/environment-variables#Option-3-CYPRESS_